docs_source_files/content/webdriver/web_element.en.md

```diff
@@ -610,7 +610,11 @@ Returns a boolean value, **True** if referenced element is
 
 {{< code-tab >}}
   {{< code-panel language="java" >}}
-// Please raise a pr code sample
+ //navigates to url 
+ driver.get("https://the-internet.herokuapp.com/checkboxes");
+  
+ //returns true if element is selected else returns false
+ boolean value =  driver.findElement(By.xpath("//*[@id="checkboxes"]/input[1]")).isSelected();
```

Comment:
```suggestion
 boolean value =  driver.findElement(By.cssSelector("input[type='checkbox']:first-of-type")).isSelected();
```

Can we use cssSelector instead of xpath. Also please change in all language bindings